com.unity.perception/Runtime/GroundTruth/Labelers/CameraLabeler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    [Serializable]
+    public abstract class CameraLabeler
+    {
+        public bool enabled = true;
+        public bool foldout = true;
+
+        internal bool isInitialized { get; private set; }
```

Comment:
Protected members are for use in derived types, whereas internal members are only for use by other types inside the Perception package, including PerceptionCamera. It locks down the type so that any old user can't "drive" these objects.